9.secretstore/README.md

```diff
@@ -0,0 +1,148 @@
+    ```
+    kubectl create secret generic mysecret --from-file ./secret
+    ```
+3. You can check that the secret is indeed stored in the Kubernetes secret store by running the command:
+    ```
+    kubectl get secret mysecret -o yaml
+    ```
+   You can see the output as below where the secret is stored in the secret store in Base64 encoded format
+   ```
+    % kubectl get secret mysecret -o yaml
+        apiVersion: v1
+        data:
+        secret: eHl6OTg3Ngo=
+        kind: Secret
+        metadata:
+        creationTimestamp: "2020-05-20T20:20:09Z"
+        name: mysecret
+        namespace: default
+        resourceVersion: "2031800"
+        selfLink: /api/v1/namespaces/default/secrets/mysecret
+        uid: 64c60c3e-dce4-4c02-b5c1-8d4dbb7cbb8f
+        type: Opaque
+    ```
+
+
+## Step 3 - Deploy the Node.js App with the Dapr Sidecar
+
+```
+kubectl apply -f ./deploy/node.yaml
+```
+
+This will deploy our Node.js app to Kubernetes.
+
+You'll also see the container image that we're deploying. If you want to update the code and deploy a new image, see **Next Steps** section. 
+
+This deployment provisions an External IP.
+Wait until the IP is visible: (may take a few minutes)
+
+```
+kubectl get svc nodeapp
+```
+
+> Note: Minikube users cannot see the external IP. Instead, you can use `minikube service [service_name]` to access loadbalancer without external IP.
+
+Once you have an external IP, save it.
+You can also export it to a variable:
+
+```
+Linux/MacOS
+export NODE_APP=$(kubectl get svc nodeapp --output 'jsonpath={.status.loadBalancer.ingress[0].ip}')
+
+Windows
+for /f "delims=" %a in ('kubectl get svc nodeapp --output 'jsonpath={.status.loadBalancer.ingress[0].ip}') do @set NODE_APP=%a
+```
+
+## Step 4 - Access the secret
+Make a request to the node app to fetch the secret. You can use the command below:
+```
+curl -k http://$NODE_APP/getsecret 
+```
+The output should be your base64 encoded secret
+
+## Step 5 - Observe Logs
+
+Now that we have our Node.js, let's watch messages come through.
+
+Get the logs of our Node.js app:
+
+```
+kubectl logs --selector=app=node -c node
+```
+
+If all went well, you should see logs like this:
+
+```
+% kubectl logs --selector=app=node -c node
+Node App listening on port 3000!
+Fetching URL: http://localhost:3500/v1.0/secrets/kubernetes/mysecret?metadata.namespace=default
+Base64 encoded secret is: eHl6OTg3Ngo=
+```
+
+In these logs, you can see that the node app is making a request to dapr to fetch the secret from the secret store. Note: mysecret is the secret that you created in Step 2
+
+## Step 6 - Cleanup
+
+Once you're done using the sample, you can spin down your Kubernetes resources by navigating to the `./deploy` directory and running:
+
+```bash
+kubectl delete -f ./deploy/node.yaml
+```
+
+This will spin down the node app.
+
+## Next Steps
```

Comment:
I really like the next steps section! 
Please lets also add a "Related links" section with a bulleted list to the relevant "how-to" area of sections, secret store concepts and anything else that makes sense here.

Reply:
fixed